e2e/bitrise.yml

```diff
@@ -86,6 +86,7 @@ workflows:
         - package_name: $PACKAGE_NAME
         - app_path: $BITRISE_DEPLOY_DIR/Wearable-release-bitrise-signed.aab|$BITRISE_DEPLOY_DIR/Application-release-bitrise-signed.aab
         - track: $TRACK
+        - mapping_file: $BITRISE_DEPLOY_DIR/Wearable-mapping.txt|$BITRISE_DEPLOY_DIR/Application-mapping.txt
```

Comment:
NOTE: We could decide to support only newline separator, this can be parsed by stepconf package (see 	example at https://github.com/bitrise-steplib/steps-git-clone/blob/d10e4259c5d6dbc92b6f816a7af99247d96420cc/gitclone/gitclone.go#L32). (It does not trim extra spaces or leave out empty lines by default tough)

Reply:
There is an existing implementation in this step (for other inputs) that handle both, that's why I decided to use the same for this input too